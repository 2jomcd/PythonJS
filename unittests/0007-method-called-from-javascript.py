with javascript:
    def call_callback(callback):
    	with python:
    		callback('one', 'two', 'three')


class SuperClass:

    def callback(self, foo, bar, baz):
        print foo
        print bar
        print baz
        print "viva l'algerie"

call_callback(SuperClass().callback)
