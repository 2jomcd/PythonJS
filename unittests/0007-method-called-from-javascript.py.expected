one
two
three
viva l'algerie
